src/components/auth/Signup.jsx

```diff
@@ -0,0 +1,80 @@
+import auth from "./auth.module.css";
+import { Link } from "react-router-dom";
+
+export function Signup() {
+  return (
+    <>
+      <main className="flex-center">
+        <div
+          className={`${auth["signup-card"]} ${auth["auth-card"]} flex-col flex-center gap-md`}
+        >
+          <h3 className={auth["auth-heading"]}>Sign up</h3>
+
+          <label className={`flex-col gap-sm ${auth["full-width"]}`}>
```

Comment:
Just a suggestion - you can create a component for elements that follow the common structure like first name, last name, email id, and reuse it.